aws/resource_aws_cognito_identity_pool.go

```diff
@@ -164,10 +170,11 @@ func resourceAwsCognitoIdentityPoolRead(d *schema.ResourceData, meta interface{}
 		AccountID: meta.(*AWSClient).accountid,
 		Resource:  fmt.Sprintf("identitypool/%s", d.Id()),
 	}
-	d.Set("arn", arn.String())
-	d.Set("identity_pool_name", ip.IdentityPoolName)
-	d.Set("allow_unauthenticated_identities", ip.AllowUnauthenticatedIdentities)
-	d.Set("developer_provider_name", ip.DeveloperProviderName)
+	_ = d.Set("arn", arn.String())
```

Comment:
We prefer to leave out the explicit notation of error checking omissions for simple uses of `d.Set()` 

```suggestion
	d.Set("arn", arn.String())
```

Same with the below changes from the previous code.

Reply:
fixed in 4ed6457ff